src/types/item.ts

```diff
@@ -1,3 +1,28 @@
+export interface EquipmentStats {
+  attackStab: number;
+  attackSlash: number;
+  attackCrush: number;
+
+  attackMagic: number;
+  attackRanged: number;
+
+  defendStab: number;
+  defendSlash: number;
+  defendCrush: number;
+
+  defendMagic: number;
+  defendRanged: number;
+
+  str: number;
+  rangedStr: number;
```

Comment:
minor: `str` is the only shortened word: Any downsides to use the full `strength` instead?